### PR TITLE
test(prices): rewrite Price List E2E Tests

### LINKED ISSUE #3350: Rewrite Price List End to End Tests
In keeping with the theme of updating tests (https://github.com/IMA-WorldHealth/bhima-2.X/pull/3346, https://github.com/IMA-WorldHealth/bhima-2.X/pull/3338), the Price List module needs it's tests to be updated.  We haven't run into any particular problems with them, but they are non-specific and hard to read since they use indexes for the ui-grid rows.  They are partway transformed into Page Object syntax, we should go ahead and go the rest of the way.

### PR DESCRIPTION
**Pull request by jniles**
This commit rewrites the price list end to end tests following the page object format.  It also tidies up the HTML a bit throughout the page, including using headercrumbs correctly and bhFilterToggle for toggling
the inline filter on the page.

Closes #3350.

### CODE REVIEW

**Comment by jniles** [comment]
@jeremielodi, can I get a review?

File: client/src/modules/prices/modal/create.html [hunk 1]
```diff
@@ -1,39 +1,42 @@
 <form name="CreateForm" bh-submit="$ctrl.submit(CreateForm)" novalidate>
+
 <div class="modal-header">
-  <h4 ng-if="$ctrl.isCreate" translate>PRICE_LIST.ADD_PRICE_LIST</h4>
-  <h4 ng-if="!$ctrl.isCreate" translate>PRICE_LIST.UPDATE</h4>
+  <ol class="headercrumb">
```
Comment (jeremielodi) [inline client/src/modules/prices/modal/create.html:4]: 👍 nice

File: client/src/modules/prices/modal/create.html [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
 
 <div class="modal-footer">
-  <bh-loading-button loading-state="CreateForm.$loading" id="submit-priceList">
-    <span translate> FORM.BUTTONS.SAVE </span>
-  </bh-loading-button>
   <button type="button" class="btn btn-default" ng-click="$ctrl.close()" translate>
     FORM.BUTTONS.CLOSE
   </button>
+
+  <bh-loading-button loading-state="CreateForm.$loading">
```
Comment (jeremielodi) [inline client/src/modules/prices/modal/create.html:37]: good use of ```bh-loading-button```

File: client/src/modules/prices/modal/createItems.html [hunk 3]
```diff
@@ -50,32 +52,37 @@ <h5 class="text-primary">{{ ModalCtrl.pricelist.label }}</h5>
...
-      <div class="col-lg-12">
-        <div class="pull-right">
-          <bh-loading-button loading-state="ModalForm.$loading" id="submit-price-list">
-            <span translate>FORM.BUTTONS.ADD</span>
-          </bh-loading-button>
-        </div>
+
+    <div class="row text-right" style="margin-bottom:15px;">
+      <div class="col-md-12">
+        <bh-loading-button loading-state="ModalForm.$loading">
```
Comment (jeremielodi) [inline client/src/modules/prices/modal/createItems.html:58]: 👍 This is useful when the internet is slow, ```bh-loading-button ``` prevent to submit twice

File: client/src/modules/prices/modal/createItems.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ function PriceListItemsModalController(
   vm.submit = submit;
   vm.cancel = cancel;
   // remove a price list item
-  vm.del = del;
+  vm.remove = remove;
```
Comment (jeremielodi) [inline client/src/modules/prices/modal/createItems.js:26]: This is more readable

File: client/src/modules/prices/templates/delete_price_item.tmpl.html [hunk 5]
```diff
@@ -1,6 +1,5 @@
-<div class="ui-grid-cell-contents text-center">
-  <a href class="text-danger" ng-click="grid.appScope.del(row.entity.uuid)">
+<div class="ui-grid-cell-contents text-right" data-row="{{row.entity.label}}">
```
Comment (jeremielodi) [inline client/src/modules/prices/templates/delete_price_item.tmpl.html:1]: good indicators for e2e test

File: test/integration/weekEndConfig.js [hunk 6]
```diff
@@ -75,7 +75,7 @@ describe('(/payroll/weekend_configuration) The /payroll/weekend_configuration  A
   });
 
   it('DELETE /WEEKEND__CONFIG/:ID will send back a 404 if the WeekEnd Configuration does not exist', () => {
-    return agent.delete('/weekend_config/inknowWeekEnd')
+    return agent.delete('/weekend_config/unknownWeekEnd')
```
Comment (jeremielodi) [inline test/integration/weekEndConfig.js:78]: 👍

**Review by jeremielodi — COMMENTED** [review]
Looks good to me

**Pull request merged**